Frameworks/UIKit/UIButton.mm

```diff
@@ -80,6 +80,16 @@ @implementation UIButton {
     ComPtr<IInspectable> _inspectableAdjustsWhenHighlightedBrush;
 }
 
+static UIEdgeInsets _decodeUIEdgeInsets(NSCoder* coder, NSString* key) {
+    id object = [coder decodeObjectForKey:key];
+    UIEdgeInsets* rawInsets = (UIEdgeInsets*)((char *)[object bytes] + 1);
```

Comment:
Apparently that's how the data is packaged; this is similar to code in UIView and elsewhere for unpacking data in the nib.

---
In reply to: [107229478](https://github.com/Microsoft/WinObjC/pull/2269#discussion_r107229478) [](ancestors = 107229478)